node/sc/sub_bridge_handler.go

```diff
@@ -49,7 +49,7 @@ type SubBridgeHandler struct {
 	chainTxPeriod         uint64
 
 	latestTxCountAddedBlockNumber uint64
-	txCountEnabledBlockNumber     uint64
+	txCountStartingBlockNumber    uint64
```

Comment:
It seems that `latestTxCountAddedBlockNumber+1` == `txCountStartingBlockNumber`.